src/cryptenroll/cryptenroll.c

```diff
@@ -474,6 +513,40 @@ static int check_for_homed(struct crypt_device *cd) {
         return 0;
 }
 
+static int prepare_luks_keyfile(
```

Comment:
hmm, this is a bit too generic. "prepare" can mean anything. And we use that verb already too often.

also, this should make clear this is about unlocking.

maybe load_unlock_keyfile()?